x-pack/plugins/observability/public/typings/data_handler/index.d.ts

```diff
@@ -25,9 +33,49 @@ interface Series {
 interface FetchDataResponse {
   title: string;
   appLink: string;
-  stats: Stat[];
   series: Series[];
 }
+
+interface LogsFetchDataResponse extends FetchDataResponse {
+  stats: {
+    [logName: string]: Numeral;
+  };
```

Comment:
Shouldn't this also contain an optional color?
```suggestion
  stats: Record<string,{
    value: number;
    color?: string;
  }>
```

Reply:
Yes, thanks 👍🏻